### ISSUE TITLE
Increase default precision in `Qks` function

### ISSUE THREAD

**Issue by multiwavelength** [Gabinou/2DKS#1]
Thank you for making your 2D KS implementation code public!

**TL;DR:** For many applications, it would be helpful to decrease the default `prec` parameter in the `Qks` function to a value below 1e-17, instead of the current 1e-06.

In the field that I work in (astronomy), it is common to report significance in terms of the number of standard deviations away from the mean of a Normal distribution. A significance level of at least 5σ is standard practice for rejecting null hypotheses. For example, a 5σ significance correspond to a p-value of 5.733e-07. 

`2DKS` calculates the equivalent p-value for the 2D KS test in the `Qks` function. The default precision in the convergence criterion is 1e-06, which means that the function exits and reports a p-value of 0 for equivalent significances above ~4.6σ. For many physics and astronomy applications, it would be very helpful to decrease the default `prec` parameter to a lower value.

I tested the code with a range of input values and the function `Qks` converges well with the default 100 iterations and prec=1e-17. This would ensure `2DKS` reports accurate p-values even for very high significance scenarios.

Thank you!

**Comment by Gabinou** [comment]
Thank you for your suggestion!
I have just pushed the suggested change.
Glad to know this library is useful in astronomy.
